pkg/services/sqlstore/user.go

```diff
@@ -818,6 +822,97 @@ func UserDeletions() []string {
 	return deletes
 }
 
+func UserUpdates() []string {
```

Comment:
Is not being called in this PR, can it be only included in the next one?

Reply:
i will remove all of these entries here. they should not be here